tests/test_graphical_units/test_threed.py

```diff
@@ -132,3 +132,28 @@ def param_trig(u, v):
     )
     trig_plane.set_fill_by_value(axes=axes, colors=[BLUE, GREEN, YELLOW, ORANGE, RED])
     scene.add(axes, trig_plane)
+
+
+@frames_comparison(base_scene=ThreeDScene)
+def test_Z_Direction(scene):
```

Comment:
Sorry, I meant test for one direction other than z, since it's already tested in the test above it. 😅

Reply:
Alright, one second.